### ISSUE TITLE
Use the linker

### ISSUE THREAD

**Issue by polluks** [geos64128/geos-desktop2.1-master#1]
Don't include the stubs
https://github.com/polluks/geos-desktop2.1-master/blob/f4670f68951798400dbea24d82d45fb3863b916d/src/desktop.c#L298

**Comment by xlar54** [comment]
Yep, understood.  Just organzing for the moment - will shift to separate .o files and link soon. Thanks!

**Comment by xlar54** [comment]
fixed per your PR.  I moved a few things around that didnt need to be extern'ed.  Thanks for the help

**Comment by xlar54** [comment]
Had a build issue that was saying "Segment 'some geos function' does not exist" in the linker.  Couldnt identify the issue.  Read somewhere that Macintosh linefeeds might cause it.  found a couple of files that had them, resaved them with Windows CRLF, but error persisted.  cvt file would be created, but it would not copy / expand.  Reverted back to include files for the moment so I can get some new changes in.  Will revisit, as I agree with your approach.

**Comment by polluks** [comment]
I see. Git should handle the EOL:
https://docs.github.com/en/github/getting-started-with-github/configuring-git-to-handle-line-endings
